infrastructure/ansible/roles/traffic_opsdb/tasks/initialize_traffic_opsdb.yml

```diff
@@ -34,9 +34,27 @@
     ssl_rootcert: "{{ postgresql_certs_ca }}"
   no_log: true
 
+- name: Create Traffic Vault Database User
+  postgresql_user:
+    encrypted: yes
+    name: "{{ tvdb_username }}"
+    password: "{{ tvdb_password }}"
+    port: "{{ postgresql_port }}"
+    login_host: 127.0.0.1
+    role_attr_flags: SUPERUSER,LOGIN,CREATEDB
```

Comment:
Does this user need to be able to delete other databases or create new ones?

Reply:
I don't think so. According to db/admin.go it looks like the superuser is used to create and drop the traffic_vault DB. So remove the `SUPERUSER` and `CREATEDB` flags?

Reply:
Ideally yes.  If some other user creates the DB and assigns ownership to the TVDB user, it shouldn't need anything more than LOGIN with regards to priv flags.

Reply:
done